nengo/builder/connection.py

```diff
@@ -80,7 +80,11 @@ def build_decoders(model, conn, rng, transform):
         targets = multiply(targets, transform.T)
 
     try:
-        wrapped_solver = model.decoder_cache.wrap_solver(solve_for_decoders)
+        if model.seeded[conn]:
```

Comment:
Oh yeah, I see now. I thought the one line in `builder/network.py` was for all networks, but it's just for the top-level. Then all networks act like objects. Okay, this makes sense.